boards/airbourne/airbourne_board.cpp

```diff
@@ -331,6 +333,36 @@ GNSSRaw AirbourneBoard::gnss_raw_read()
   return raw;
 }
 
+bool AirbourneBoard::battery_voltage_present() const
+{
+  return this->battery_monitor_.has_voltage_sense();
+}
+
+float AirbourneBoard::battery_voltage_read() const
+{
+  return static_cast<float>(this->battery_monitor_.read_voltage());
+}
+
+void AirbourneBoard::battery_voltage_set_multiplier(double multiplier)
```

Comment:
Does this multiplier need to happen in the board layer? Should this be managed in the firmware sensors module?